src/com/java/class22/GitPractice.java

```diff
@@ -2,9 +2,16 @@
 
 public class GitPractice {
     public static void main(String[] args) {
+
         System.out.println("This is from Chirag's PC");
         System.out.println("gitKnowsMe was here");
         System.out.println("TestingMyFirstProgram");
+
+        System.out.println("TestingHomeFeature-Swati");
+        System.out.println("Testing");
+        System.out.println("Login Feature Started");
+        System.out.println("Login feature completed till teh time manager is coming!! yay!");
```

Comment:
what do you mean by the manager is coming????? it doesn't looks good so remove it.